### PR TITLE
[WIP] 1-1 chats test group

### LINKED ISSUE #13041: Refactor 1-1 chat group
Currently contains 11 tests to merge.
Testrail: https://ethstatus.testrail.net/index.php?/suites/view/48&group_by=cases:section_id&group_order=asc&display_deleted_cases=0&group_id=50655 (1-1 chat)

### PR DESCRIPTION
**Pull request by yevh-berdnyk**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

fixes #13041 

status: wip

### CODE REVIEW

File: test/appium/tests/atomic/chats/test_one_to_one.py [hunk 1]
```diff
@@ -1,131 +1,412 @@
...
+        [device.click_system_home_button() for device in (self.device_1, self.device_2)]
+        [device.open_notification_bar() for device in (self.device_1, self.device_2)]
+        if (self.device_2.element_by_text_part(message).is_element_displayed()
+                or self.device_1.element_by_text_part(emoji_unicode).is_element_displayed()):
             self.errors.append("PN are keep staying after message was seen by user")
         self.errors.verify_no_errors()
 
+    @marks.testrail_id(6305)
+    @marks.critical
+    def test_image_in_one_to_one_send_save_reply_timeline(self):
```
Comment (churik) [inline test/appium/tests/atomic/chats/test_one_to_one.py:272]: IMO better to remove from here check for timeline and have it separately (as far as I understand it is not a prio for now, so it is fine to have them in medium, i.e. as a part of test_different_status_in_timeline)

File: test/appium/tests/atomic/chats/test_one_to_one.py [hunk 2]
```diff
@@ -782,46 +889,40 @@ def test_markdown_support_in_messages(self):
...
+        self.chat.send_message(message)
+        self.home.just_fyi("Deleting chat via delete button and check it will not reappear after relaunching app")
+        self.chat.delete_chat()
+        self.chat.get_back_to_home_view()
+
+        if self.home.get_chat_from_home_view(self.chat_name).is_element_displayed():
+            self.errors.append('Deleted %s chat is shown, but the chat has been deleted' % self.chat_name)
+        self.driver.close_app()
+        self.driver.launch_app()
+        self.sign_in.sign_in()
```
Comment (churik) [inline test/appium/tests/atomic/chats/test_one_to_one.py]: self.home.reopen_app()

Comment (churik) [reply test/appium/tests/atomic/chats/test_one_to_one.py]: Also don't think it is worth to start a separate group only for 1 test, maybe better to add this as the last test into the 2driver group or move it to medium

File: test/appium/tests/atomic/chats/test_public.py [hunk 3]
```diff
@@ -297,6 +299,42 @@ def test_public_navigate_to_chat_when_relaunch(self):
         if not self.chat.chat_element_by_text(text_message).is_element_displayed(30):
             self.drivers[0].fail("Not navigated to chat view after reopening app")
 
+    @marks.testrail_id(5317)
+    @marks.critical
+    def test_public_copy_and_paste_messages(self):
+        message_text = {'text_message': 'mmmeowesage_text'}
+        formatted_message = {'message_with_link': 'https://status.im',
```
Comment (churik) [inline test/appium/tests/atomic/chats/test_public.py:336]: flaky test in general; as sometimes message as a button can't be tapped (happens from time to time, tried to tap by coordinates, but didn't help). Perhaps it is better  to move to medium, WDYT?

**Review by churik — APPROVED** [review]

File: test/appium/tests/atomic/chats/test_one_to_one.py (hunk 1 above)
Comment (yevh-berdnyk) [reply test/appium/tests/atomic/chats/test_one_to_one.py:272]: Done

File: test/appium/tests/atomic/chats/test_one_to_one.py (hunk 2 above)
Comment (yevh-berdnyk) [reply test/appium/tests/atomic/chats/test_one_to_one.py]: I think we can move other tests to this group too. But it's better not to have tests with 1 driver in the same group with multiple drivers.

File: test/appium/tests/atomic/chats/test_public.py (hunk 3 above)
Comment (yevh-berdnyk) [reply test/appium/tests/atomic/chats/test_public.py:336]: Agree, moved it

**Comment by churik** [comment]
copy in https://github.com/status-im/status-react/pull/13229

**Pull request closed without merging**